tools/github_readme_sync/tests/hierarchy_test.py

```diff
@@ -178,10 +179,8 @@ def run_server():
             f.write(f"[Fragment]({self.server_url}/valid#fragment)\n")
 
         with self.assertLogs(level="ERROR") as log:
-            try:
+            with contextlib.suppress(SystemExit):
```

Comment:
oh this is cool!

Reply:
Yeah, Ruff told me to do it https://docs.astral.sh/ruff/rules/suppressible-exception/